benchmarks/run_benchmarks.py

```diff
@@ -289,9 +302,7 @@
             if verbose:
                 print(cmd, file=log_file, flush=True)
 
-            p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
-                    universal_newlines=True)
-            out, err = p.communicate()
+            p, out, err, _ = run_process(cmd)
```

Comment:
Please verify that this usage is appropriate for the blocks starting on line 315 and 327. As far as I can tell, the behavior should be unmodified.

Reply:
I agree, this should be correct as the `timeit` or `pyperf` modules take care of timing